packages/ui/src/components/GalleryPost/GalleryPost.tsx

```diff
@@ -53,89 +65,149 @@ export default function GalleryPost({
     [inlines, textPostIsJustLinkedImage]
   );
 
+  const handleImagePressed = useCallback(
+    (uri: string) => {
+      onPressImage?.(post, uri);
+    },
+    [onPressImage, post]
+  );
+
   if (!postIsJustImage && !postIsJustText && !postIsJustReference) {
     return null;
   }
 
   return (
-    <Pressable onPress={onPress} onLongPress={() => onLongPress?.(post)}>
+    <Pressable
+      onPress={() => onPress?.(post)}
+      onLongPress={() => onLongPress?.(post)}
+    >
       <View padding="$m" key={post.id} position="relative" alignItems="center">
-        {(postIsJustImage || textPostIsJustLinkedImage) && (
-          <ImageBackground
-            source={{ uri: postIsJustImage ? image!.src : linkedImage }}
-            style={{
-              width: HEIGHT_AND_WIDTH,
-              height: HEIGHT_AND_WIDTH,
-            }}
-            imageStyle={{
-              borderRadius: 12,
-            }}
-          >
-            <View position="absolute" bottom="$l">
-              <GalleryAuthorRow
-                author={post.author}
-                authorId={post.authorId}
-                sent={post.sentAt}
-                width={HEIGHT_AND_WIDTH}
-              />
-            </View>
-          </ImageBackground>
-        )}
+        {(postIsJustImage || textPostIsJustLinkedImage) &&
+          (detailView ? (
+            <TouchableOpacity
+              onPress={() =>
+                handleImagePressed(postIsJustImage ? image!.src : linkedImage!)
+              }
+              activeOpacity={0.9}
```

Comment:
Are we using similar opacity values elsewhere? Might make sense to extract as a constant.

Reply:
It just looks better at this opacity imo, but we don't use it elsewhere. I can just leave it out for now.